package.json

```diff
@@ -23,16 +23,19 @@
 		"commit": "git-cz",
 		"cz": "git-cz"
 	},
-	"dependencies": {},
+	"dependencies": {
+		"@klasa/ratelimits": "^0.1.0",
+		"@klasa/utils": "^0.1.0"
+	},
 	"devDependencies": {
 		"@commitlint/cli": "^9.1.1",
 		"@commitlint/config-conventional": "^9.1.1",
 		"@klasa/core": "github:dirigeants/core#build",
 		"@skyra/eslint-config": "^5.2.1",
 		"@types/jest": "^26.0.5",
 		"@types/node": "^14.0.22",
-		"@typescript-eslint/eslint-plugin": "^3.6.0",
-		"@typescript-eslint/parser": "^3.6.0",
+		"@typescript-eslint/eslint-plugin": "^3.7.0",
+		"@typescript-eslint/parser": "^3.7.0",
```

Comment:
please don't upgrade packages, let dependabot do that

Reply:
What Dependabot? We use Depfu and it updates deps every week.

Reply:
If you check git blame on those lines you can see that they were committed by Favna.

Reply:
@kyranet whatever I didn't check (I think this still uses dependabot tho)

Reply:
I didn't update them per-say, I installed new dependencies which might have led to them being updated. Does it really matter?